src/run.ts

```diff
@@ -17,9 +17,19 @@ import resolveFrom from "resolve-from";
 
 const createRelease = async (
   octokit: ReturnType<typeof github.getOctokit>,
-  { pkg, tagName }: { pkg: Package; tagName: string }
+  { pkg, tagName }: { pkg: Package; tagName: string },
+  options: {
+    createGithubReleases?: boolean;
+  } = {}
 ) => {
   try {
+    if (
+      options.createGithubReleases !== undefined &&
+      !options.createGithubReleases
+    ) {
+      return;
+    }
```

Comment:
I still kinda think that this should be moved outside of the `createRelease` function - if I call such a function I expect it to actually do what it is named after. Kinda a single-responsibility approach.

Reply:
Updated!